site/content/chat-widget-moments/moments-in-livechat.md

```diff
@@ -4,4 +4,4 @@ weight: 40
 
 # Moments in LiveChat
 
-Moments can be triggered only by [rich message's](https://www.livechatinc.com/kb/rich-messages/) button. Currently, LiveChat only supports rich messages delivered by chatbots using our [ChatBot](https://chatbot.com) product. However, we are still working on other possibilities to provide rich messages to chat, such as including them inside Agent App Widgets or by using backend Chat APIs.
+Moments can be triggered by [rich message's](https://www.livechatinc.com/kb/rich-messages/) buttons or using [MessageBox](https://developers.livechatinc.com/docs/agent-app-widgets/#messagebox) integration in Agent App. Currently, LiveChat only supports rich messages delivered by chatbots using our [ChatBot](https://chatbot.com) product. However, we are still working on other possibilities to provide rich messages to chat, such as including them inside Agent App Widgets or by using backend Chat APIs.
```

Comment:
Nie do końca "or". MessageBox integration też wysyła RM button. To zdanie bym zostawił, ale edytował to co jest niżej:
> Currently, LiveChat only supports rich messages delivered by chatbots using our [ChatBot](
To jest teraz nieprawdą